### PR TITLE
[svs-staking] SVS Staking strategy

### PR DESCRIPTION
**Pull request by fsjuhl**
Combines normal Vampires and staked vampires.
For https://twitter.com/SVSNFT

### CODE REVIEW

File: src/strategies/svs-staking/index.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    }],
+    "name": "getVampsBuried",
+    "outputs":[{
+        "internalType": "uint256[]",
+        "name": "",
+        "type": "uint256[]"
+    }],
+    "stateMutability": "view",
+    "type": "function"
+}];
```
Comment (ChaituVR) [inline src/strategies/svs-staking/index.ts:21]:
```suggestion
const stakingAbi = [ 'function getVampsBuried(address burier) view returns (uint256[])' ];
```

File: src/strategies/svs-staking/index.ts [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    }],
+    "name":"balanceOf",
+    "outputs":[{
+        "internalType":"uint256",
+        "name":"",
+        "type":"uint256"
+    }],
+    "stateMutability":"view",
+    "type":"function"
+ }];
```
Comment (ChaituVR) [inline src/strategies/svs-staking/index.ts:37]:
```suggestion
const tokenAbi = [ 'function balanceOf(address owner) view returns (uint256)' ];
```

**Comment by fsjuhl** [comment]
How do i merge this haha?

**Review by ChaituVR — APPROVED** [review]

**Comment by ChaituVR** [comment]
Strategy is live: https://snapshot.org/#/strategy/svs-staking

**Pull request merged**